.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "C_Cpp.default.compileCommands": "${workspaceFolder}/build/compile_commands.json"
```

Comment:
For at dette skal være nyttig trenger jeg også

```cmake
set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```
i CMakeLists.txt!

Reply:
Whoops, glemte den. Takk!